testsupport/tiers/checks.go

```diff
@@ -48,333 +47,234 @@ var (
-		clusterResourceQuotaConfigMap(),
-		numberOfClusterResourceQuotas(),
-		idlers(518400, "dev", "stage"))
 }
 
-type basedeactivationdisabledTierChecks struct {
-	baseTierChecks
-}
-
-func (a *basedeactivationdisabledTierChecks) GetTierObjectChecks() []tierObjectCheck {
-	return []tierObjectCheck{nsTemplateTier(a.tierName, 0)}
+func getClusterObjectChecks(tierName string) func() []clusterObjectsCheck {
+	return func() []clusterObjectsCheck {
+		switch tierName {
+		case appstudio:
+			return clusterObjectsChecks(
+				clusterResourceQuotaCompute(cpuLimit, "1750m", "7Gi", "15Gi"),
+				clusterResourceQuotaDeployments(),
+				clusterResourceQuotaReplicas(),
+				clusterResourceQuotaRoutes(),
+				clusterResourceQuotaJobs(),
+				clusterResourceQuotaServices(),
+				clusterResourceQuotaBuildConfig(),
+				clusterResourceQuotaSecrets(),
+				clusterResourceQuotaConfigMap(),
+				numberOfClusterResourceQuotas(),
+				idlers(43200, ""))
+		case baselarge:
+			return clusterObjectsChecks(
+				clusterResourceQuotaCompute(cpuLimit, "1750m", "16Gi", "15Gi"),
+				clusterResourceQuotaDeployments(),
+				clusterResourceQuotaReplicas(),
+				clusterResourceQuotaRoutes(),
+				clusterResourceQuotaJobs(),
+				clusterResourceQuotaServices(),
+				clusterResourceQuotaBuildConfig(),
+				clusterResourceQuotaSecrets(),
+				clusterResourceQuotaConfigMap(),
+				numberOfClusterResourceQuotas(),
+				idlers(43200, "dev", "stage"))
+		case baseextendedidling:
+			return clusterObjectsChecks(
+				clusterResourceQuotaCompute(cpuLimit, "1750m", "7Gi", "15Gi"),
+				clusterResourceQuotaDeployments(),
+				clusterResourceQuotaReplicas(),
+				clusterResourceQuotaRoutes(),
+				clusterResourceQuotaJobs(),
+				clusterResourceQuotaServices(),
+				clusterResourceQuotaBuildConfig(),
+				clusterResourceQuotaSecrets(),
+				clusterResourceQuotaConfigMap(),
+				numberOfClusterResourceQuotas(),
+				idlers(518400, "dev", "stage"))
+		case base1ns:
+			return clusterObjectsChecks(
+				clusterResourceQuotaCompute(cpuLimit, "1750m", "7Gi", "15Gi"),
+				clusterResourceQuotaDeployments(),
+				clusterResourceQuotaReplicas(),
+				clusterResourceQuotaRoutes(),
+				clusterResourceQuotaJobs(),
+				clusterResourceQuotaServices(),
+				clusterResourceQuotaBuildConfig(),
+				clusterResourceQuotaSecrets(),
+				clusterResourceQuotaConfigMap(),
+				numberOfClusterResourceQuotas(),
+				idlers(43200, "dev"))
+		case advanced:
+			return clusterObjectsChecks(
+				clusterResourceQuotaCompute(cpuLimit, "1750m", "16Gi", "15Gi"),
+				clusterResourceQuotaDeployments(),
+				clusterResourceQuotaReplicas(),
+				clusterResourceQuotaRoutes(),
+				clusterResourceQuotaJobs(),
+				clusterResourceQuotaServices(),
+				clusterResourceQuotaBuildConfig(),
+				clusterResourceQuotaSecrets(),
+				clusterResourceQuotaConfigMap(),
+				numberOfClusterResourceQuotas(),
+				idlers(0, "dev", "stage"))
+		case testtier:
+			return []clusterObjectsCheck{}
+		default:
+			return clusterObjectsChecks(
```

Comment:
I think it's safer for the default to be an error so that the tests must be explicit.

Reply:
done in a013cb3